src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -223,15 +240,10 @@ const ProteinsListItemInfo = (props: Props) => {
               </div>
             )}
           </div>
-          {proteinSummaryStats &&
-            uniprotXref &&
+          { proteinSummaryStats &&
```

Comment:
A very minor issue but noticed there is this unnecessary space between the opening curly brace and the variable